notebooks/sippy-analysis.ipynb

```diff
@@ -0,0 +1,2411 @@
+{
```

Comment:
<p>A lot of the above is just about getting access to the data which is covered in a notebook, can this be streamlined to only what is needed for you to get the data into your notebook?</p><p> </p><p>I think what is important about this notebook starts below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>